lib/spring/test/acceptance_test.rb

```diff
@@ -83,14 +83,14 @@ def without_gem(name)
       end
 
       test "help message when called without arguments" do
-        assert_success "bin/spring", stdout: 'Usage: spring COMMAND [ARGS]'
+        assert_success "bundle exec bin/spring", stdout: 'Usage: spring COMMAND [ARGS]'
```

Comment:
`with_clean_env` feels like we are secretly adding `bundle exec` to every command.
However, `with_clean_env` is lot cleaner and gets the job done. Also, issue of conflicting bundler version seems to be only specific for travis runs. Locally, I have bundler 1.10.6 and I don't get any error.